src/stringProcessing/relevantWords.js

```diff
@@ -308,9 +309,43 @@ function getRelevantWords( text, locale ) {
 	return take( combinations, relevantWordLimit );
 }
 
+/**
+ * Gets relevant words from keyphrase and synonyms, metadescription, and subheadings.
+ *
+ * @param {string} keyphrase The keyphrase of the paper.
+ * @param {string} synonyms The synonyms of the paper.
+ * @param {string} metadescription The metadescription of the paper.
+ * @param {string[]} subheadings The subheadings of the paper.
+ * @param {string} locale The locale of the paper.
+ *
+ * @returns {WordCombination[]} Relevant word combinations from the paper attributes.
+ */
+function getRelevantWordsFromTopic( keyphrase, synonyms, metadescription, subheadings, locale ) {
```

Comment:
I think `getRelevantWordsFromTopic` is not a good description of this method.
Perhaps `getRelevantWordsFromPaperAttributes`?
(Although subheadings are not necessarily attributes / metadata).

Reply:
Since we could add new metadata / paper attributes in the future, it could be worthwhile to put the attribute arguments in this function in a single object argument:
```js
/**
 * Gets relevant words from the paper's attributes.
 * 
 * @param {Object} attributes                  The attributes to process
 * @param {string} attributes.keyphrase        The keyphrase of the paper.
 * @param {string} attributes.synonyms         The synonyms of the paper.
 * @param {string} attributes.metadescription  The metadescription of the paper.
 * @param {string[]} attributes.subheadings    The subheadings of the paper.
 * @param {string} locale                      The locale of the paper.
 *
 * @returns {WordCombination[]} Relevant word combinations from the paper attributes.
 */
function getRelevantWordsFromPaperAttributes( attributes, locale ) {
      const { keyphrase, synonyms, metadescription, subheadings } = attributes;
```
(And the parameter descriptions can be aligned as well 😛(going full Igor here) ).

Reply:
Never go full Igor 😜